### ISSUE TITLE
Wolfenstein 3D - stuttering

### ISSUE THREAD

**Issue by dukeblooders** [schellingb/dosbox-pure#323]
Hi,

Can't get Wolf3D or Blake Stone to run properly. Menus work correctly, but once the game starts, it's unplayable.

I downloaded the Steam version, which works and includes a dosbox config and I try to use it with my setup.
The file is detected but the result is the same.

Any idea ?

Here's my pure config and the dosbox file :
[config.zip](https://github.com/schellingb/dosbox-pure/files/9227140/config.zip)

**Comment by schellingb** [comment]
Games like DOOM and most made after it run in something called "protected mode" while most games made before, like Wolfenstein and Blake Stone, run in "real mode". Because many old DOS games made in the 80s aren't able to detect the CPU speed and make their game logic run at the same speed everywhere, DOSBox ends up running all real mode games at a default CPU speed of a rather old 286 CPU. This is too slow for most real mode games made in the 90s...

DOSBox Pure in version 0.23 introduced a built-in database of game release years to combat this issue (see #48 and #97). It tries to detect the games release year and scales the CPU speed to something appropriate for when the game was released. 

I assume the release date of the games aren't detected properly because the Steam versions are just a bit different from what I used to build the game database. If you can provide me a list of files (file names and file sizes) of these versions, I can expand the game detection to include those versions. That would be great because many people have the Steam versions.

For now you need to go to the core options into `Performance Options` and set `Emulated Performance` to something like a 486 or even MAX (because these games correctly react on the CPU speed). If this doesn't solve the issue, we need to look deeper :-)

**Comment by dukeblooders** [comment]
Thanks for your answer.

I tried to change Emulated Performance (486DX1, DX2 and Max), without success (Max increases the framerate but it's still laggy).
I also tried different versions of Wolf3D before trying the Steam version, so I'm not sure it's an issue with that version (+ I use a non-Steam version of Black Stone).

![Files](https://user-images.githubusercontent.com/48026962/182013964-a6e84c7f-a1cd-487e-a392-ed03ed54058d.png)
